taqueria-sdk/index.ts

```diff
@@ -73,6 +75,18 @@ export const execCmd = (cmd: string): LikeAPromise<StdIO, ExecException> =>
 		});
 	});
 
+export const spawnCmd = (fullCmd: CmdArgEnv) =>
+	new Promise((resolve, reject) => {
+		const cmd = fullCmd[0];
+		const args = fullCmd[1];
+		const envVars = fullCmd[2];
+		const child = spawn(cmd, args, { env: { ...process.env, ...envVars }, stdio: 'inherit' });
+		child.on('close', code => {
```

Comment:
After seeing your implementation, I see now why you had the success msgs. I was thinking of just doing:
```
child.on('close', resolve); // leave it up to the caller to decide what to do.
child.on('error', reject);
```

The above is technically correct as different programs might use exit codes that have significant meaning, such as https://tldp.org/LDP/abs/html/exitcodes.html

Although atypical, non-zero exit codes doesn't necessarily mean that there was a failure.